docs/csharp/tuples.md

```diff
@@ -133,7 +133,7 @@ names `Item1`, whose value is `42` and `stringContent`, whose value is "The answ
 
 There are two conditions where candidate field names are not projected onto the tuple field:
 
-1. When the candidate name is a reserved tuple name. Examples include `Item3`, `ToString`. or `Rest`.
+1. When the candidate name is a reserved tuple name. Examples include `Item3`, `ToString` or `Rest`.
```

Comment:
I think it was meant to be a comma

```suggestion
1. When the candidate name is a reserved tuple name. Examples include `Item3`, `ToString`, or `Rest`.
```

Reply:
I made the suggested changes. Thanks @Youssef1313.